Menu

kennetvu opened the pull request.

Commit:
renamed all stories to .jsx files

Commit:
PR cleanup

Commit:
Added two new menu components, sublist and itemLink. Updated menu story with correct components

Commit:
fixed lint problems for all stories

Commit:
done with menu

Commit:
removed local state in menu

Commit:
Code cleanup and removed unused dependencies

alexanbj commented:
After thinking about this, I may have an idea for for we can drop the Menu.Sublist component. Mind if I make some modifications to this?

Commit:
Cleanup Menu component

By using context, we can get rid of the SubMenuList component.
This allows for nesting the MenuList component, and the component itself
knows whether to apply the css class or not

alexanbj commented:
Closing this. Fixed directly in master
